ast/parser_ext.go

```diff
@@ -565,9 +572,13 @@ func (a commentKey) Compare(other commentKey) int {
 
 // ParseStatements returns a slice of parsed statements.
 // This is the default return value from the parser.
-func ParseStatements(filename, input string) ([]Statement, []*Comment, error) {
+func ParseStatements(filename, input string, processAnnotation ...bool) ([]Statement, []*Comment, error) {
```

Comment:
Nice! This way we can introduce the option w/o breaking existing callers. One change I'd like to see:

Instead of `func(string, string, ...bool)` make it `func(string, string, ...ParserOption)`  and then define the ParserOption type to support multiple options. For now, we'll only have the one (processAnnotation) but in future we could have more.

Reply:
Done 7a14b61